page-object/pom.xml

```diff
@@ -39,8 +39,7 @@
     </dependency>
     <dependency>
       <groupId>org.junit.jupiter</groupId>
-      <artifactId>junit-jupiter</artifactId>
-      <version>RELEASE</version>
+      <artifactId>junit-jupiter-api</artifactId>
       <scope>test</scope>
     </dependency>
```

Comment:
You can remove the whole dependency because it is not used anymore.

Reply:
Nice catch 👍